k8_cortx_cloud/cortx-cloud-helm-pkg/cortx-ha/templates/cortx-ha-pod.yaml

```diff
@@ -0,0 +1,184 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: {{ .Values.cortxha.name }}
+  name: {{ .Values.cortxha.name }}
+  namespace: {{ .Values.namespace }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: {{ .Values.cortxha.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.cortxha.name }}
+        {{- if .Values.cortxha.machineid.value }}
+        cortx.io/machine-id: {{ .Values.cortxha.machineid.value }}
+        {{- end }}
+        name: cortx-ha
+        cortx.io/service-type: cortx-ha
+    spec:
+      hostname: {{ .Values.cortxha.service.headless.name }}
+      serviceAccountName: {{ .Values.cortxha.serviceaccountname }}
+      automountServiceAccountToken: true
```

Comment:
I updated this to resolve the need for the SA token, as defined by https://jts.seagate.com/browse/UDX-6849